frontend/src/components/core/Block/Block.tsx

```diff
@@ -251,13 +252,14 @@ class Block extends PureComponent<Props> {
       plotlyChart: (el: SimpleElement) => (
         <PlotlyChart element={el} width={width} />
       ),
-      progress: (el: SimpleElement) => (
-        <Progress
-          value={el.get("value")}
-          className="stProgress"
-          style={{ width }}
-        />
-      ),
+      progress: (el: SimpleElement) =>
+        isStale ? null : (
+          <Progress
+            value={el.get("value")}
+            className="stProgress"
+            style={{ width }}
+          />
+        ),
```

Comment:
Won't this cause elements in the app to jump around?

How about this solution:
* Make the stale `stProgress` CSS not show the blue bar (i.e. make it transparent or gray)
* And also remove the CSS transition when stale, so it doesn't move backward